pizone/zone.py

```diff
@@ -88,6 +88,78 @@ def airflow_min(self) -> int:
         """Min allowed airflow for the zone as a percent"""
         return self._get_zone_state('MinAir')
 
+    async def set_airflow_min(self, value: int) -> None:
```

Comment:
We will also need a test case to cover these APIs. It would also be great if the rest of the v2 API could be filled out as well, but this is less essential.

Reply:
I don't have any v2 devices so not sure how much I can help here unfortunately.

Reply:
If it's v1, then you can test it locally. See the stuff in the test directory. I usually just played around with it and checked that it worked on the device.

Reply:
Thanks. Just doing some local testing now. Will have some results for you in the next 48 hours.